composer.json

```diff
@@ -12,7 +12,7 @@
         "ext-tokenizer": "*",
         "codeigniter/coding-standard": "^1.1",
         "codeigniter4/codeigniter4": "4.x-dev",
-        "friendsofphp/php-cs-fixer": "^3.1",
+        "friendsofphp/php-cs-fixer": "^3.40",
```

Comment:
Why do you change this?

Reply:
I made composer require in order to run the php-cs-fixer. Was it  a mistake?

Reply:
Yes, you should have use `composer update` instead of `composer require`.
Can you remove the change?

Reply:
I removed the change.
Please can you guide me in the failed tests on how to resolve them?